### PR TITLE
feat(MOPLAT-397): Add progress bar to CMS web view

### PR DESCRIPTION
**Pull request by patrinoua**
### Description

This PR resolves [MOPLAT-397]

Feedback from QA, bring the loading bar to the top 

Screen Shot 2022-08-30 at 11 21 50

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-397]: https://artsyproduct.atlassian.net/browse/MOPLAT-397?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/sharedUI/screens/Artwork/ArtworkWebView.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const ArtworkWebView = () => {
         onLoad={() => setIsProgressBarVisible(false)}
       />
       {isProgressBarVisible && (
-        <Flex px={4} height="100%" justifyContent="center">
+        <Flex px={4} height="100%">
```
Comment (rajsam003) [inline src/app/sharedUI/screens/Artwork/ArtworkWebView.tsx:24]: As per @brainbicycle's comment, if it has to match with Eigen, then we dont need padding at all
```suggestion
        <Flex height="100%">
```

**Review by rajsam003 — APPROVED** [review]
🚀

**Comment by brainbicycle** [comment]
👀 looking better! can we get rid of the padding for the header so it goes full width of the screen?

**Comment by patrinoua** [comment]
@brainbicycle I am not sure what you mean, it currently is full width (although personally I feel this is not the best UI) 
Screen Shot 2022-08-30 at 17 01 58

**Comment by brainbicycle** [comment]
sorry I was going off the screenshot in the description, if it is full width great!

**Comment by patrinoua** [comment]
Sorry @brainbicycle I forgot to update the screenshot and also thought that the below was visible so you had context, but it was marked as resolved and folded! 🙈 
Screen Shot 2022-09-01 at 14 56 36

**Pull request merged**